File: C++/13.cpp
#include <iostream>
#include <vector>
#include <string>

int main() {
    std::vector<int> v = {1, 2, 3, 4, 5};
    std::string str = "Hello World";

    std::cout << "Vector size: " << v.size() << std::endl;
    std::cout << "String size: " << str.size() << std::endl;

    return 0;
}

File: JS/13.js
// 화살표 함수
const greet = (name) => 'Hello, ' + name + '!';

// 본문이 여러 줄인 경우, 중괄호 사용
const multiply = (name, age) => {
    const birth_day = new Date().getFullYear() - age + 1;
    return 'Hello, ' + name + '! ' + birth_day;
};

console.log(greet("John")); // Hello, John!
console.log(multiply("John", 19)); // Hello, John! 2006
